projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -3,23 +3,20 @@
 ### This is a list detailing changes for all Jetpack releases.
 
 ## 11.0-a.11 - 2022-05-24
-### Enhancements
-- Added the post field to the Publicize package [#24324]
-
 ### Improved compatibility
-- Update test [#24454]
+- General: improve consistency of available modules [#24454]
```

Comment:
I think this is more of a "other" change since it's really a backend change that will not be visible for users.